test_stack.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+from __future__ import unicode_literals
+import pytest
+from stack import Stack
+
+
+@pytest.fixture()
+def create_stack():
+    return Stack([1, 2, 3])
+
+
+@pytest.fixture()
+def create_empty_stack():
+    return Stack()
+
+
+@pytest.fixture
+def create_single_item_stack():
+    return Stack([1])
+
+
+def test_pop(create_stack):
+    assert create_stack.pop() == 3
```

Comment:
Think about something for a moment.  The fact that you are testing for "3" being the value returned by pop means that your test has knowledge of the nature of your fixture.  Is there some way you could think of to help break the need for that special knowledge?  What happens to your test if you decide that the values in your stack should be `["foo", "bar", "baz"]`?  See what I mean?